ftplugin/python/SimpylFold.vim

```diff
@@ -5,9 +5,9 @@ let b:loaded_SimpylFold = 1
 
 let s:blank_regex = '\v^\s*(\#.*)?$'
 if &ft == 'pyrex' || &ft == 'cython'
-    let b:def_regex = '\v^\s*%(%(class|def|cdef|cpdef|ctypedef) \w+)|cdef:'
+    let b:def_regex = '\v^\s*%(%(class|def|async +def|cdef|cpdef|ctypedef) \w+)|cdef:'
```

Comment:
Why not `\s` like below? Are tabs not legal in Pyrex or Cython?

Also, I recommend changing the existing `def` cases to `%(async\s+)?def`, rather than adding a separate case.